exonum-java-binding/core/rust/src/testkit/mod.rs

```diff
@@ -213,3 +214,122 @@ fn create_java_keypair<'a>(
         &[secret_key_byte_array.into(), public_key_byte_array.into()],
     )
 }
+
+// Converts Java array of `TestKitServiceInstances` to vector of `InstanceConfig`.
+//
+// `TestKitServiceInstances` representation:
+//      String artifactId;
+//      String artifactFilename;
+//      ServiceSpec[] serviceSpecs;
```

Comment:
nit: not a question for this PR, but why do we have naming inconsistency between `Service` in Java and `Instance` in Rust?